gradle/project.gradle.kts

```diff
@@ -142,25 +145,25 @@ tasks {
         exclude("*sources.jar", "*javadoc.jar")
     }
 
-    val jre by creating {
-        val jdkLocal = workDir.resolve("jre")
+    val jdk by creating {
+        val jdkLocal = workDir.resolve("java").toPath()
```

Comment:
This breaks my .exe and the exe generator script. Pls notify me before making changes to directory structure.

Reply:
Surely. I just thought that something has to be done, because the applications says it's a JDK, and it is, but it's still called jre for some reason. Considering a future opportunity of it not needing a full JDK I resorted to call it Java.